### PR TITLE
Classes structure

### PR DESCRIPTION
**Pull request by kamranasif506**
In this PR

- Created Persorn Class
- Created student Class and inherits from Person class
- Created teacher Class and inherits from Person class

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission, name)
+    @classroom = classroom
+  end
```
Comment (anagudelogu) [inline student.rb]: - [ ] You did a great job working on this milestone! 💯 However, please make sure that you're passing the arguments to your `super` correctly:

If you look closely, your `Person` class constructor expects two named (or hashed) parameters: `name` and `parent_permission`, however, you're passing them as normal ones, please consider passing them correctly! 🤓 🔧 

```rb
  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
    super(age, parent_permission: parent_permission, name: name)
    @classroom = classroom
  end
```

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
```
Comment (anagudelogu) [inline student.rb]: - [ ] Please consider using `require_relative` when you're importing files in your own project. It's a recommended practice to keep `require` for external libraries/gems! 🤓 🔧

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+class Teacher < Person
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission, name)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```
Comment (anagudelogu) [inline teacher.rb]: - [ ] Please do the same with this one, correct the `super` method and use `require_relative`! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @kamranasif506, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @kamranasif506, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Used `require_relative` ✅ 
- Passing arguments to `super` ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**